### DISCUSSION TITLE
How do you use Facebook Provider on localhost?

### DISCUSSION THREAD

**Discussion by rickywid** [nextauthjs/next-auth#1250] [Help]
I'm trying to create a Facebook OAuth app to use with NextAuth but in the Facebook App settings it gives me this 

https://i.imgur.com/9unfmWp.png

**Comment by oliviermtl** [comment]
Just leave it blank

**Comment by rickywid** [comment]
@oliviermtl https://i.imgur.com/B9Alk7L.png

**Comment by oliviermtl** [comment]
You might want to provide a bit more details than just a screenshot.
I was doing exactly the same this morning, works out of the box.

**Comment by rickywid** [comment]
@oliviermtl what info do you need?

I'm simply providing the Facebook client and secret id in the provider config settings. Not sure what else I have to do to get it to work.

**Comment by theniallriver** [comment]
It should work on localhost without providing facebook with an authorized callback url. What happens when you try and login using Facebook on localhost? @rickywid
